src/include/sof/audio/format_hifi3.h

```diff
@@ -53,9 +53,23 @@ static inline ae_int32x2 vec_sat_int32x2(int64_t x, int64_t y)
 	/* Round and saturate both 64-bit values to 32-bit and pack them */
 	return (ae_int32x2)AE_ROUND32X2F64SSYM(d0, d1);
 }
+
```

Comment:
Nit about commit text, the new version isn't more accurate, it's mote efficient.